examples/create-react-app/src/App.js

```diff
@@ -0,0 +1,74 @@
+import { Routes, Route, Link } from 'react-router-dom'
+import { Provider as RollbarProvider, ErrorBoundary, RollbarContext, useRollbar } from '@rollbar/react'
+
+import './App.css';
+
+const rollbarConfig = {
+  accessToken: process.env.REACT_APP_PUBLIC_ROLLBAR_TOKEN,
+  hostSafeList: [
+    'localhost:3000',
+    'localhost:4000',
+  ],
+  captureUncaught: true,
+  captureUnhandledRejections: true,
+  payload: {
+    environment: 'development',
+    client: {
+      javascript: {
+        code_version: '1.0.0',
+        source_map_enabled: true,
+      }
+    }
+  }
+};
+
+function App() {
+  return (
+    <RollbarProvider config={rollbarConfig}>
+      <ErrorBoundary>
+        <RollbarContext context="whole-app">
+          <nav>
+            <ul>
+              <li>
+                <Link to="/a">A</Link>
+              </li>
+              <li>
+                <Link to="/b">B</Link>
+              </li>
+            </ul>
+          </nav>
+          <Router />
+        </RollbarContext>
+      </ErrorBoundary>
+    </RollbarProvider>
+  );
+}
+
+export default App;
+
+function Router() {
+  return (
+    <Routes>
+      <Route path="a" element={<RouteA />} />
+      <Route path="b" element={<RouteB />} />
```

Comment:
Instead of the whole-app context above, why not wrap each route:
```
<RollbarContext context="/a">
  <Route path="a" element={<RouteA />} />
</RollbarContext>
<RollbarContext context="/b">
  <Route path="b" element={<RouteB />} />
</RollbarContext>
```